### ISSUE TITLE
Allow module options to be passed as ['nuxt-security', options]

### ISSUE THREAD

**Issue by Pierre-M** [Baroshem/nuxt-security#87]
### Version

nuxt-security: 0.10.1
nuxt: 3.1.0

### Steps to reproduce

In you Nuxt config, add the `nuxt-security` module as a `[moduleName, moduleOptions]` entry.

```typescript
export default defineNuxtConfig({
  modules: [
    ["nuxt-security", {headers: {...}}]
  ]
})
```

### What is Expected?

The passed configuration should be taken into account.

### What is actually happening?

The module options are only handled if they are passed through the `security` configuration key in Nuxt config.

### Possible fix

In the module definition, instead of using explicitly `nuxt.options.security` to retrieve passed options, you should use `options directly` : 

```typescript
{
    setup(options, nuxt) {
    // ...
    nuxt.options.security = defuReplaceArray(options, {
      ...defaultSecurityConfig
    });
}
```

Thanks for you module, it's a really huge and valuable work 🤗

**Comment by Baroshem** [comment]
Hey, thanks for reporting that!

I will take a look at it in the upcoming days :)

Also, I am glad that you like it!

**Comment by Baroshem** [comment]
Thanks to the PR by @runyasak this functionality is now merged to main. I will work on few other functionalities this week and will release a new version with a support for that feature.

Stay tuned! :)

**Comment by Baroshem** [comment]
Closing the issue as the functionality was merged and released as 0.10.2 :)
